projects/core/src/global-message/http-interceptors/handlers/unknown-error/unknown-error.handler.ts

```diff
@@ -20,9 +21,10 @@ import { HttpErrorHandler } from '../http-error.handler';
   providedIn: 'root',
 })
 export class UnknownErrorHandler extends HttpErrorHandler {
-  responseStatus = HttpResponseStatus.UNKNOWN;
-
   protected logger = inject(LoggerService);
+  protected featureConfigService = inject(FeatureConfigService);
```

Comment:
[MEDIUM]
```suggestion
  private featureConfigService = inject(FeatureConfigService);
```

see coding guidelines [link](https://wiki.one.int.sap/wiki/display/spar/Coding+Guidelines#CodingGuidelines-UsingFeatureTogglesinTypeScript)